### PR TITLE
WIP: upgrade to IPFS v0.27

### LINKED ISSUE #5: Update to latest ipfs
Same as https://github.com/ipfs-shipyard/peerpad/issues/125

### PR DESCRIPTION
**Pull request by pgte**
Trying to fix #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,19 +7,20 @@
     "build": "browserify src/index.js -d -o public/index.bundle.js",
     "build:watch": "watchify src/index.js -d -o public/index.bundle.js -v",
     "start": "npm run build && npm run start:server",
-    "start:server": "serve public"
+    "start:server": "serve public",
+    "update:ipfs": "wget https://unpkg.com/ipfs/dist/index.min.js -O public/ipfs.bundle.js"
```
Comment (daviddias) [inline package.json]: ah, right, if I don't republish, all those bug fixes don't get to this version!

Comment (daviddias) [reply package.json]: publishing a patch version :)

**Review by daviddias — CHANGES_REQUESTED** [review]
You just imported two bundles (277K lines of code). It shouldn't be necessary. Anything specific that you are trying to achieve here? 

Also, ignore the package-lock.json

File: package.json (hunk 1 above)
Comment (daviddias) [reply package.json]: 0.27.1 released. You should be golden now :)

**Comment by pgte** [comment]
@diasdavid 2 bundles? You mean the app and the js-ipfs bundle? If so, that was because of the modest workshop, so people wouldn't have to go through installing js-ipfs..

**Comment by daviddias** [comment]
@pgte understood. Can't the app require from unpkg directly?

**Comment by pgte** [comment]
We were relying on conference Wi-Fi, but yes, the we could do that :)

**Pull request merged**